app/controllers/api/v1/api_docs_controller.rb

```diff
@@ -58,12 +58,12 @@ class ApiDocsController < ActionController::Base
           key :description, 'Study publication/data link operations'
         end
         tag do
-          key :name, 'MetadataSchemas'
-          key :description, 'Metadata Convention schema definitions'
+          key :name, 'Bookmarks'
+          key :description, 'Bookmarked/saved views for users'
         end
         tag do
-          key :name, 'Reports'
-          key :description, 'Reports/Portal Stats'
```

Comment:
Can we keep these endpoints?  

They're a bit buggy right now, but were essential for some [times-series analytics not in Mixpanel](https://docs.google.com/spreadsheets/d/1FSpP2XTrG9FqAqD9X-BHxkCZae9vxZA3cQLow8mn-bk/edit#gid=0).  Not deleting them would make the fix easier.  (If we do delete them, we should probably do it holistically.)

Reply:
I took them out of the main Swagger controller because (while listed here) the referenced controller/endpoints were never "swaggerized", so they couldn't actually be used via Swagger.  You can confirm this by looking at [production](https://singlecell.broadinstitute.org/single_cell/api/v1#/Reports) right now.  If we want them added in, that's totally fine, but out-of-scope for this work.

Reply:
Ah, thanks for helping me understanding!  These changes make sense.